### PR TITLE
Feature/use remote schemas

### PR DESCRIPTION
**Pull request by southeo**

### CODE REVIEW

File: src/main/resources/json-schema/annotation-processing-request.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      "pattern": "^https:\/\/hdl\\.handle\\.net\/[\\w.]+\/(.){3}-(.){3}-(.){3}",
+      "examples": [
+        "https://hdl.handle.net/20.5000.1025/XXX-XXX-XXX"
+      ]
+    },
+    "oa:motivation": {
+      "description": "The motivation for the annotation. Based on a selection of https://www.w3.org/TR/annotation-model/#motivation-and-purpose. The motivation ods:adding is added for DiSSCo's purposes.",
+      "enum": [
+        "ods:adding",
+        "ods:deleting",
```
Comment (TomDijkema) [inline src/main/resources/json-schema/annotation-processing-request.json:34]: This needs to be tombstoning no?

File: src/main/resources/json-schema/annotation.json [hunk 2]
```diff
@@ -367,6 +181,7 @@
     "ods:version",
     "oa:motivation",
     "oa:hasTarget",
+    "oa:hasBody",
```
Comment (TomDijkema) [inline src/main/resources/json-schema/annotation.json:184]: Nice!

**Review by TomDijkema — DISMISSED** [review]
One question, overall looks very good! Love the refs to other schemas!

File: src/main/resources/json-schema/annotation-processing-request.json (hunk 1 above)
Comment (southeo) [reply src/main/resources/json-schema/annotation-processing-request.json:34]: this is automatically pulled from [schemas.dissco,tech](https://schemas.dissco.tech/schemas/fdo-type/annotation/0.3.0/annotation.json), it's ods:deleted there

**Pull request merged**